src/menus/ViewMenus.cpp

```diff
@@ -127,33 +127,44 @@ void DoZoomFitV(AudacityProject &project)
    auto &tracks = TrackList::Get( project );
 
    // Only nonminimized audio tracks will be resized
-   auto range = tracks.Any<AudioTrack>()
+   auto resizableTracks = tracks.Any<AudioTrack>()
       - [](const Track *pTrack){
          return TrackView::Get( *pTrack ).GetMinimized(); };
-   auto count = range.size();
-   if (count == 0)
+
+   // Bug 2803: Cast to int, because otherwise the result of 
+   // division will be unsigned too, and will be a very large number
+   // if height will be negative!
+   auto resizableTracksCount = static_cast<int>(resizableTracks.size());
+
+   if (resizableTracksCount == 0)
       return;
 
    // Find total height to apportion
    auto height = viewInfo.GetHeight();
-   height -= 28;
```

Comment:
Why was this line lost?

The old behavior left a bit of background space at the bottom (if possible, consistent with minimum height constraints).  This is convenient for the user to click on to deselect all tracks, and there will be other convenient context menu items.

Are you losing this part of the old behavior?

Reply:
Removed this during debugging, it was easier to check that everything fits as supposed without this gap. There are still empty space below this track which can be accessed by scrolling, but agree that this is unneccessary change in behaviour, will fix that